posts/2024/2024_07_29_kaustav_week_9.rst

```diff
@@ -0,0 +1,65 @@
+My Journey Continues: Week 9 Progress with DIPY
```

Comment:
Hi Kaustav!

Great Job and good luck with your PhD applications.

See some comments below

Reply:
Thank you.